server/core/src/rodsConnect.cpp

```diff
@@ -558,10 +558,10 @@ printZoneInfo() {
 
     /* print the reHost */
     if ( getReHost( &tmpRodsServerHost ) >= 0 ) {
-        log::server::info({{"re_host", tmpRodsServerHost->hostName->name}});
+        log::server::info({{"re_host", tmpRodsServerHost->hostName->name}}); // TODO Is this referring to the delay server?
     }
     else {
-        log::server::info({{"re_host", "error"}});
+        log::server::info({{"re_host", "error"}}); // TODO Is this referring to the delay server?
```

Comment:
same.    whether the delay server is / should be running is now going to be controlled only by the database setting, not a per-server server_config.json value.

Reply:
I'll rename them to be `delay_server_host`. When the delay server migration code is merged, I can use `irods::get_delay_server_pid()` to determine this information. That's assuming the delay server is up by then. If that doesn't work, I can try and make a database call to get that information.

Reply:
After looking at the code, I think we should remove these and expose a new subcommand via iadmin, `get_delay_server`. That subcommand can print the leader and successor as JSON giving the admin a way to react to migrations, etc.

Reply:
interesting.    so a new general admin call?

Reply:
Yes. Since I removed those properties, admins won't have a way to determine who is running the delay server. This new subcommand gives them a very easy way to get that information. And it's even easier to implement than `set_delay_server` because all of the work has been done.

I thought about exposing the R_GRID_CONFIGURATION table via GenQuery, but that felt wrong because that table may be used to hold sensitive information that normal users shouldn't be able to see.

Reply:
agreed.   we talked through the genquery option earlier and came to the same conclusion re: security.